src/views/company-invitations.njk

```diff
@@ -58,6 +58,21 @@
         window.addEventListener('focus', () => {
             location.reload();
         });
+
+        let text = "{{ids}}";
+        let idArray = text.split(",");
```

Comment:
This is a bit unclear at a glance, as the back-end has `ids` as a list of string ids, but now it seems to be a comma separated string of ids - I think this is because Nunjucks is implicitly calling the `toString` before rendering the list, which converts it to a comma separated string:


```javascript
['a', 'b', 'c'].toString() // 'a,b,c'
```

This can be a little dangerous as it looks like the end of the ids have `invitedBy` email addresses (based on the test data):

https://github.com/companieshouse/your-companies-web/blob/ee72fe74b052318d67ec3191022cb86183955b3b/test/mocks/associations.mock.ts#L141

Email addresses can have commas so splitting on it could be error prone in those scenarios:

```javascript
['1-2-a@a.a', '1-2-b@b.b', '1-2-c,c@c.c'].toString().split(',')
// ['1-2-a@a.a', '1-2-b@b.b', '1-2-c', 'c@c.c']
```

I'd suggest dumping the array and then escaping it (to prevent any XSS from user emails) - note I haven't tested this as I don't have my local env running quite yet, so this is only theory:

```suggestion
        let text = '{{ ids | dump | escape }}';
        let idArray = JSON.parse(text);
```

Reply:
The comma thing might be overkill as-well, while you can technically have a comma inside of an email, most public providers prevent it, also [GOVUK One Login](https://signin.account.gov.uk/enter-email-create) - where I'm guessing these emails are being ingested from, doesn't allow it...